onnxscript/rewriter/_rewrite_rule.py

```diff
@@ -531,14 +531,22 @@ def _apply_to_graph_or_function(
                     f = ir.Function(domain, name, overload, graph=graph, attributes=())
                     model.functions[f.identifier()] = f
 
-                # If we are fusing nodes, update the metadata props of the new node(s)
+                # If we are fusing nodes, update the docstring of the new node(s)
+                attributes = ["namespace", "pkg.torch.onnx.class_hierarchy", "pkg.torch.onnx.fx_node", "pkg.torch.onnx.name_scopes", "pkg.torch.onnx.stack_trace"]
```

Comment:
For class_hierarchy and name_scopes, we should parse the info and find the common leading scopes.